main.go

```diff
@@ -888,6 +889,10 @@ func main() {
 		options.CFlags = strings.Split(*cFlags, " ")
 	}
 
+	if *cxxFlags != "" {
+		options.CXXFlags = strings.Split(*cxxFlags, " ")
```

Comment:
I plan on removing `options.CFlags` (because it's not necessary anymore with better CGo support) so I think it's better to not add it here for this PR. It's a bit inconsistent but not a problem in my opinion.

Reply:
Maybe problematic: 

In the good old days with larger C/C++ codebases, CPPFLAGS was used for flags common to C and C++ , using CFLAGS for only C specific Flags and CXXFLAGS. However I wanted to have a shortcut implementing C++ support, as I wrote previously. So I hesitated from implementing CPPFLAGS and changing all target json files. 

Not sure how to proceed. My gut feeling is to have code duplicated now and rework later if this really is limiting tinygo, though not exactly clean code.